### ISSUE TITLE
[GitHub] Deprecation of Basic Auth

### ISSUE THREAD

**Issue by gr2m** [semantic-release/cli#326]
If you use the semantic-release CLI to setup `semantic-release` in a GitHub repository, then you probably receive an E-Mail from GitHub with the subject "[GitHub] Deprecation Notice", stating that 

> You recently used a password to access an endpoint through the GitHub API...

We are aware of the deprecation. I'm working on simple OAuth Web Flow as an alternative

**Comment by Xunnamius** [comment]
I think this is finally busted? I can't use `npx semantic-release-cli init` with GitHub Actions at all, even using `--gh-token`. I get `Error: no auth mechanism defined` and cannot continue without selecting "other" and doing the remaining setup manually.

**Comment by gr2m** [comment]
> I think this is finally busted?

yup. See https://github.com/semantic-release/cli/issues/345 for updates
